tower-http/src/classify/status_in_range_is_error.rs

```diff
@@ -4,34 +4,6 @@ use std::{fmt, ops::RangeInclusive};
 
 /// Response classifier that considers responses with a status code within some range to be
 /// failures.
-///
-/// # Example
-///
-/// A client with tracing where server errors _and_ client errors are considered failures.
```

Comment:
Why did you remove this example?

Reply:
I could find a way to make a hyper client that implemented `tower::Service` but I see now that hyper-util provides one! I've brought this example and the one in `lib.rs` back.